.github/workflows/multus.yaml

```diff
@@ -0,0 +1,67 @@
+name: Multus integration tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    paths:
+      - cmd/userfacing/**
```

Comment:
It would be interesting to use these PR paths on more workflows, then we wouldn't need the skip_ci flag as much if the integration tests have a good idea when to trigger.

Reply:
Yeah! It might be nice to **force** a CI test to run just in case we didn't have all the right paths set for it, but I suppose that can be done manually with the "Actions" tab I think?